src/app/main-api-handler.ts

```diff
@@ -171,6 +171,9 @@ ipcMain.on(apiName.symphonyApi, async (event: Electron.IpcMainEvent, arg: IApiAr
             break;
         case apiCmds.setCloudConfig:
             const { podLevelEntitlements, acpFeatureLevelEntitlements, pmpEntitlements, ...rest } = arg.cloudConfig as ICloudConfig;
+            if (podLevelEntitlements && podLevelEntitlements.autoLaunchPath && podLevelEntitlements.autoLaunchPath.match(/\\\\/g)) {
```

Comment:
what happens if we have six backslashes instead of four that we have today?

Reply:
@VishwasShashidhar No matter how many `\\\\` there are in the ACP input this check makes sure that it will replace to single escaped backslash characters 

![2020-03-26 13 11 07](https://user-images.githubusercontent.com/13243259/77622009-87dbdd00-6f63-11ea-9931-47eae62012be.gif)